2019-robot.iml

```diff
@@ -12,11 +12,11 @@
     <orderEntry type="module-library">
       <library>
         <CLASSES>
-          <root url="jar://$USER_HOME$/.gradle/caches/modules-2/files-2.1/com.kauailabs.navx.frc/navx-java/3.1.344/30981bb5cf08864cfdca75cfd21418470e16c167/navx-java-3.1.344.jar!/"/>
+          <root url="jar://C:/Users/NathanDesjardins/.gradle/caches/modules-2/files-2.1/com.kauailabs.navx.frc/navx-java/3.1.344/30981bb5cf08864cfdca75cfd21418470e16c167/navx-java-3.1.344.jar!/"/>
         </CLASSES>
```

Comment:
You need to be careful of commiting the IML file.  In general you should only commit it if you made substantive changes, and then you have to be sure it references $USER_HOME$ not your own user directory.

Reply:
Sorry, that seemed to happen automatically for some reason. I'll fix that right now and push it.